src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/electric/machines/ElectricGoldPan.java

```diff
@@ -40,31 +40,28 @@ public class ElectricGoldPan extends AContainer implements RecipeDisplayItem {
     private final GoldPan goldPan = SlimefunItems.GOLD_PAN.getItem(GoldPan.class);
     private final GoldPan netherGoldPan = SlimefunItems.NETHER_GOLD_PAN.getItem(GoldPan.class);
 
-    private final ItemStack gravel = new ItemStack(Material.GRAVEL);
-    private final ItemStack soulSand = new ItemStack(Material.SOUL_SAND);
-
     @ParametersAreNonnullByDefault
     public ElectricGoldPan(ItemGroup itemGroup, SlimefunItemStack item, RecipeType recipeType, ItemStack[] recipe) {
         super(itemGroup, item, recipeType, recipe);
         addItemSetting(overrideOutputLimit);
     }
 
     /**
-     * This returns whether the {@link ElectricGoldPan} will stop proccessing inputs
+     * This returns whether the {@link ElectricGoldPan} will stop processing inputs
      * if both output slots contain items or if that default behavior should be
-     * overriden and allow the {@link ElectricGoldPan} to continue processing inputs
+     * overridden and allow the {@link ElectricGoldPan} to continue processing inputs
      * even if both output slots are occupied. Note this option will allow players
      * to force specific outputs from the {@link ElectricGoldPan} but can be
      * necessary when a server has disabled cargo networks.
      * 
-     * @return If output limits are overriden
+     * @return If output limits are overridden
      */
-    public boolean isOutputLimitOverriden() {
```

Comment:
This is a public API, we would need a deprecation period for the old name

Reply:
should i have for removal set to true? or leave it as just Deprecated